openslides_backend/action/actions/meeting_user/update.py

```diff
@@ -13,6 +16,26 @@ class MeetingUserUpdate(UpdateAction):
 
     model = MeetingUser()
     schema = DefaultSchema(MeetingUser()).get_update_schema(
-        optional_properties=["comment"],
+        optional_properties=[
+            "comment",
+            "number",
+            "structure_level",
+            "about_me",
+            "vote_weight",
+        ],
     )
     permission = Permissions.User.CAN_MANAGE
+
+    def check_permissions(self, instance: Dict[str, Any]) -> None:
+        if "about_me" in instance and not any(
+            field in instance
+            for field in ("comment", "number", "structure_level", "vote_weight")
```

Comment:
s.o.

Reply:
Changed the condition into` len(instance) ==1`.